pandas/tests/io/test_spss.py

```diff
@@ -71,3 +73,13 @@ def test_spss_usecols(datapath):
 
     with pytest.raises(TypeError, match="usecols must be list-like."):
         pd.read_spss(fname, usecols="VAR00002")
+
+
+def test_can_read_spss_file_from_local_path_as_string(datapath):
+    sav_path = datapath("io", "data", "spss", "labelled-num.sav")
+    pd.read_spss(sav_path)
+
+
+def test_can_read_spss_file_from_local_path_as_pathlib_path(datapath):
```

Comment:
instead of this, I would just parameterize test_spss_labelled_num like

```
@pytest.mark.parameterize('path_klass', [lambda p: p, Path])
```

and wrap datapath